doc/source/cluster/cluster_under_construction/key-concepts.rst

```diff
@@ -1,130 +1,78 @@
---------------
+The autoscaler is a process that runs on the :ref:`head node<cluster-head-node-under-construction>`.
+It is responsible for provisioning or deprovisioning :ref:`worker nodes<cluster-worker-nodes-under-construction>`
+to meet the needs of the Ray workload. In particular, if the resource demands of the Ray workload exceed the
+current capacity of the cluster, the autoscaler will attempt to add more nodes. Conversely, if
+a node is idle for long enough, the autoscaler will remove it from the cluster.
 
-Ray Job submission is a mechanism to submit locally developed and tested applications
-to a remote Ray cluster. It simplifies the experience of packaging, deploying,
-and managing a Ray application. To learn more about Ray jobs, refer to the
-:ref:`documentation<ray-job-submission-api-ref>`.
+To learn more about the autoscaler and how to configure it, refer to the following user guides:
 
-Cloud clusters
---------------
+* :ref:`Configuring Autoscaling on VMs<deployment-guide-autoscaler-under-construction>`.
+* :ref:`Configuring Autoscaling on Kubernetes<kuberay-index>` (TODO cade@ update this link.)
 
-If you’re using AWS, GCP, Azure (community-maintained) or Aliyun (community-maintained), you can use the
-:ref:`Ray cluster launcher<cluster-launcher>` to launch cloud clusters, which
-greatly simplifies the cluster setup process.
+.. _cluster-clients-and-jobs-under-construction:
 
-Cluster managers
+Clients and Jobs
 ----------------
+Ray provides two methods for running workloads on a Ray Cluster: the Ray Client, and Ray Job Submission.
 
-You can simplify the process of managing Ray clusters using a number of popular
-cluster managers including :ref:`Kubernetes<kuberay-index>`,
-:ref:`YARN<ray-yarn-deploy>`, :ref:`Slurm<ray-slurm-deploy>` and :ref:`LSF<ray-LSF-deploy>`.
+* **The Ray Client** enables interactive development by connecting a local Python script or shell to the cluster.
+  Developers can scale-out their local programs on the cloud as if it were on their laptop. The Ray Client is used
+  by specifying the :ref:`head node<cluster-head-node-under-construction>` address as an argument to `ray.init`.
+* **Ray Job Submission** enables users to submit locally developed-and-tested applications to a remote Ray
+  Cluster. Ray Job Submission simplifies the experience of packaging, deploying, and managing a Ray application.
 
-Kubernetes (K8s) operator
--------------------------
+To learn how to run workloads on a Ray Cluster, refer to the following user guides:
 
-Deployments of Ray on Kubernetes are managed by the Ray Kubernetes Operator. The
-Ray Operator makes it easy to deploy clusters of Ray pods within a Kubernetes
-cluster. To learn more about the K8s operator, refer to
-the :ref:`documentation<kuberay-index>`.
+* :ref:`Running Ray workloads on VMs<ref-cluster-quick-start-vms-under-construction>` (TODO cade@ update this link).
+* :ref:`Running Ray workloads on Kubernetes<kuberay-index>` (TODO cade@ update this link).
```

Comment:
There's a quickstart guide that can be linked `<kuberay-quickstart>`.
Linking the K8s landing page <kuberay-index> is ok to.